examples/network-hhmodel/loader.rs

```diff
@@ -0,0 +1,85 @@
+use std::path::Path;
+
+use csv::Reader;
+use ixa::context::Context;
+use ixa::define_person_property;
+use ixa::{ContextPeopleExt, PersonId};
+use serde::Deserialize;
+use std::fs::File;
+
+#[derive(Deserialize, Copy, Clone, PartialEq, Eq, Debug, Hash)]
+pub enum AgeGroupValue {
+    U5,
+    U18,
+    Adult,
+    Old,
+}
+
+#[derive(Deserialize, Copy, Clone, PartialEq, Eq, Debug, Hash)]
+pub enum SexValue {
+    Female,
+    Male,
+}
+
+#[derive(Deserialize, Debug)]
+struct PeopleRecord {
+    id: u16,
+    age_group: AgeGroupValue,
+    sex: SexValue,
+    household_id: u16,
+}
+
+define_person_property!(Id, u16);
+define_person_property!(AgeGroup, AgeGroupValue);
+define_person_property!(Sex, SexValue);
+define_person_property!(HouseholdId, u16);
```

Comment:
I would consider putting each of these right after the respective structs, but that's a taste issue.